data-scripts/_generators/SimpleJapaneseListGenerator.ts

```diff
@@ -0,0 +1,41 @@
+// @ts-ignore
+import Kuroshiro from 'kuroshiro'
+// @ts-ignore
+import KuromojiAnalyzer from 'kuroshiro-analyzer-kuromoji'
+import SimpleListGenerator from './SimpleListGenerator'
+
+export default class SimpleJapaneseListGenerator extends SimpleListGenerator {
+  async convertJapanese() {
+    const kuroshiro = new Kuroshiro()
+    await kuroshiro.init(new KuromojiAnalyzer())
+    console.info('Converting hiragana and katakana to romaji')
+    const promises = this.data.map(async (entry) => {
+      const isKatakana = entry.split('').every((char) => {
+        return Kuroshiro.Util.isKatakana(char)
+      })
+      if (isKatakana) {
+        return entry
+      }
```

Comment:
Only one more thing!
When `isKatakana === true`, let's exclude the entry from this dictionary.

Reply:
Are you sure about this? Because there are quite a lot of katakana only words shouldn't they be converted and used?

Reply:
> Are you sure about this? Because there are quite a lot of katakana only words shouldn't they be converted and used?

@MrWook Yes, I'm sure.

Currently, Katakana words (e.g. `カリフォルニア`) are not converted into Romaji, including into dictionary.
As I mentioned before, commonly, Japanese words (Hiragana, Katakana, Kanji) are [not permitted](https://github.com/zxcvbn-ts/zxcvbn/pull/119#discussion_r913350845) in password field.
So we need to avoid including Katakana words in the dictionary.

There are two options. 

-  :one: Katakana words which consist of all Katakana are also converted into Romaji
-  :two: Katakana words which consist of all Katakana are excluded from the dictionary

I do not think :one: works. IMO, Typing `arubamu` in password field seems to be weird. They use English for that case  so to deal with this, English dictionary need to be used at the same time.

```js
import { zxcvbn, zxcvbnOptions } from '@zxcvbn-ts/core'
import zxcvbnCommonPackage from '@zxcvbn-ts/language-common'
import zxcvbnEnPackage from '@zxcvbn-ts/language-en'
import zxcvbnJaPackage from '@zxcvbn-ts/language-ja'

const password = 'somePassword'
const options = {
  translations: zxcvbnJaPackage.translations,
  graphs: zxcvbnCommonPackage.adjacencyGraphs,
  dictionary: {
    ...zxcvbnCommonPackage.dictionary,
    ...zxcvbnEnPackage.dictionary,
    ...zxcvbnJaPackage.dictionary,
  },
}
```

The :two: option might be better to deal with Katakana word.